DataFormats/BTauReco/src/TaggingVariable.cc

```diff
@@ -127,7 +127,7 @@ const char* const TaggingVariableTokens[] = {
   /* [trackPtRatio]                             = */ "trackPtRatio",
   /* [trackPParRatio]                           = */ "trackPParRatio",
 
-  /* [trackIp2dVal]                             = */ "trackIp2dVal",
+  /* [trackIp2dSig]                             = */ "trackIp2dSig",
```

Comment:
Sorry, Mauro, I was a bit too quick with my comment. Yes, you are completely right, there is indeed an inconsistency in the TaggingVariable code. I suggest you simply remove all instances of either `trackIp2dVal` or `trackIp2dVal` since neither of them seems to be used anywhere. Also, can you also rename `trackJetDist` to `trackJetDistVal` in `TaggingVariableTokens`.

Concerning the unordered map, I guess it will require a long initialization list in the constructor where the same type of mistakes and typos can still happen. So unless that would speed up the code, I would not bother with redesigning this part of the code.